package.json

```diff
@@ -466,7 +466,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
-    "@folio/stripes-acq-components": "~5.0.0",
+    "@folio/stripes-acq-components": "~5.1.0",
     "@folio/stripes-data-transfer-components": "~6.0.0",
```

Comment:
Why you haven't bumped `stripes-data-transfer-components`? It's already released

Reply:
@UladzislauKutarkin Because The module was not released when I created the PR.